crypto/core_namemap.c

```diff
@@ -335,3 +303,107 @@ int ossl_namemap_add_names(OSSL_NAMEMAP *namemap, int number,
 
     return number;
 }
+
+/*-
+ * Pre-population
+ * ==============
+ */
+
+#ifndef FIPS_MODE
+#include <openssl/evp.h>
+
+/* Creates an initial namemap with names found in the legacy method db */
+static void get_legacy_evp_names(const char *main_name, const char *alias,
+                                 void *arg)
+{
+    int main_id = ossl_namemap_add_name(arg, 0, main_name);
+
+    /*
+     * We could check that the returned value is the same as main_id,
+     * but since this is a void function, there's no sane way to report
```

Comment:
This is an internal function, it need not be `void`.  The calling functions are but isn't it early enough to change their behaviour?  [ or does it cascade up the call chain ]

Reply:
This is downstream from a callback for OBJ_NAME_do_all(), so the returned value is ignored either way...

Reply:
I suspected as much.  Thanks for explaining.